app-android/src/main/res/layout/fragment_school_overview.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you please add the vertical margins between fields? We have a vertical form item margin constant in the dimens.xml